compiler/test/formatter_outputs/comments.gr

```diff
@@ -3,15 +3,15 @@
  followed by a blank line
 */
 
-let myfun = (/* a */ x, /*post*/ y) => 10
+let myfun = (/* a */ x /*post*/, y) => 10
```

Comment:
Why did you change the output locations of this?

Reply:
(Same as others, decision is arbitrary, see below)

Reply:
Ah, so it's actually *better* at keeping the input location now. Nice!